crates/wysiwyg/src/dom/dom_methods.rs

```diff
@@ -440,6 +448,33 @@ where
         action_list
     }
 
+    // FIXME: this is a workaround to avoid inserting the same text twice when selection is between 2 line breaks/ZWSP nodes.
```

Comment:
Honestly, this is a very ugly workaround, but I couldn't think of any other solution that wouldn't require a whole re-write of this function. All suggestions for a better replacement are welcome.